centreon/lang/es_ES.UTF-8/LC_MESSAGES/messages.po

```diff
@@ -16172,3 +16172,12 @@ msgstr ""
 
 # msgid "The limit of 2 units has been reached"
```

Comment:
```suggestion
# msgid "You have reached the limit of 2 units"
```

Reply:
Done in this upcoming PR: https://github.com/centreon/centreon/pull/2029/files#diff-dc289dcc37d296608a03cf9223221a68a496ee20a5ab3e83d37bb2a202040112